src/datasets/utils/_hf_hub_fixes.py

```diff
@@ -42,9 +39,10 @@ def create_repo(
         `str`: URL to the newly created repo.
     """
     if version.parse(huggingface_hub.__version__) < version.parse("0.5.0"):
+        name, *organization = repo_id.split("/")
```

Comment:
if you use `*organization`, then `organization` is a list, not a string
```suggestion
        organization, name = repo_id.split("/") if "/" in repo_id else None, repo_id
```

Reply:
That is why I put below:
```python
organization=organization or None
```
because `[] or None` is `None`.

Reply:
Anyway, this method is internal and we only call it for Hub datasets (see the rest of the modified files): so organization is never None.

Reply:
Oh, I see, it is NOT a str but a list... Oops!!

Reply:
Maybe I can just ignore the case where organization is None?
```python
name, organization = repo_id.split("/")
```

Reply:
I mean we do not create/delete canonical repos with these methods...

Reply:
The only publicly available one is in `_push_parquet_shards_to_hub`, and there if there is no "/", we replace `repo_id` by preceding it with the current username as organization:
```python
        elif len(identifier) == 1:
            dataset_name = identifier[0]
            organization_or_username = api.whoami(token)["name"]
            repo_id = f"{organization_or_username}/{dataset_name}"